### DISCUSSION TITLE
Website for WeAreMahsaAmini.com

### DISCUSSION THREAD

**Discussion by syavash** [WeAreMahsaAmini/Core#15] [Ideas]
We can use Ghost or another static Markdown website builder to create a website for this event that everyone can contribute to. Github README can be a good thing for techy people but we need a good looking website as well that one can share for awareness.

**Comment by Diba-Sch** [comment]
Create a repository of English articles (from credible newsletters/journals) to provide context for non-Iranians

**Comment by Mahsa-eb47** [comment]
I can work on the UX/UI design when the project brief, target audience, and key features are ready.

**Comment by Azadi2022** [comment]
I’m a UX/UI designer and can give a hand there. Also I can translate EN to ES

**Comment by IranFreedom** [comment]
I'd say a CMS will do the job, WordPress is preferred. Most people know how to use it and it makes the initial version implementation faster

**Reply by mobidar** [reply]
> I'd say a CMS will do the job, WordPress is preferred. Most people know how to use it and it makes the initial version implementation faster

I agree. For the first version the fastest thing is Wordpress. 

I think a nice landing page with a brief of the story, a creative timeline of the events and some links to resources they can follow the news is a good MVP 

**Edited**:
Not a wordpress guy, but if you decide to go with something like Next.js/Gatsby, Contenful let me know. I've worked many projects with this stack.

**Reply by syavash** [reply]
@mobidar We are using Gatsby at this time. For the timebeing we're keeping it simple and update website through PRs. The benefit is the fact that we can work together and everything needs to get approved. We can definitely use something like Contentful or Strapi in the future.

**Comment by atriaa** [comment]
I can also help as a UX Designer. 

We can have an English section/version of the site, which is mainly for non-Iranians and its goal could be to spread awareness at this stage. We can also have a Farsi version for people inside Iran that hosts more useful information, like how to defend yourself, some strategies and tactics, VPNs, etc.

**Reply by syavash** [reply]
@atriaa The main purpose of the website is creating awareness for non-Iranians and the goal is to have the mainpage in English and have pages for other languages as well.

**Comment by nazunderground** [comment]
I've created tons of websites in the past if you guys need help with color, image placement, fonts, etc. just let me know. I have experience in graphics and design. I've used square space, wix, and tons of other platforms - so I can design really quickly. I would suggest we use a template to begin with and have tons of great imagery so its effective. Also, we need a page for a list of resources. 
thoughts??

**Reply by nazunderground** [reply]
has anyone started this project??? let me know.

**Comment by SourceMP** [comment]
I suggest these contents for the site :

- About Mahsa Amini
- Gallery ( Artwork , Video )
- Celebrities list who supported us ( with news link ,twitter link or instagram link )
- Donation
- #MahsaAmini's Hashtag Counter
- Memorial section for protestors who sacrificed their lives
- Protests day counter
- Links to pages that have contents like Medical supports , ...


Update :
- Multi-Language Contents ( Persian , English , Deutsch , Turkish , ...)
- "ways you can help" (e.g. social media activities 'Twitter, Instagram, Youtube ', ... )

**Reply by kaveh6** [reply]
since keeping all the contents up to date with all different languages could be challenging, we could at least do this for the main contents . but website should definitely support multi language

**Reply by SourceMP** [reply]
We can create a channel or group to organize content and assign data collection to someone who wants to help for the site

**Reply by Saba2nd** [reply]
Just a suggestion:
“About #MahsaAmini” (” # “ better describe the movement and clears that this is not about one individual )
“About Women Rights” ( laws, rules, and history, etc)
“Why now? “
keep the “donation” to the end (are we sure if we want to collect donation? Who is going to organize and how?)

**Reply by ghost** [reply]
I know English french translator. I will add him to this channel.

**Reply by hanabarazandeh** [reply]
I agree with all the content, it includes the celebrity wall and the art gallery (two other ideas that got a lot of votes). I think it will encourage more artists to take part because they will be credited. And it will encourage more celebrities to support too. The only thing is from what I hear there is no safe way to donate, but if that changes, it would be a great idea.

**Reply by amir-aein** [reply]
Hey all, I have a comment about the content flow in the webpage. What if we could create a linear story telling approach, getting the reader to care about the cause step by step. Here is a small draft of different steps, I'd love to hear everyone's thoughts on this:

1. Who is Mahsa Amini ⎯ build emotional connection
2. What is happening in Iran ⎯ convert the sorrow to anger
3. Why should you care ⎯ convert the anger to rage
4. How can you help ⎯ convert rage to action

In general, I think a more active tone in the content/design of the website is more suitable to what is happening. Currently it feels more sad and passive.

That is not say I don't agree with all the content you guys have suggested here, but I think a lot of could be categorized as 'resources', something that we can have at very bottom of the page.

**Comment by cygen-X** [comment]
We should have content on the website since we need to share it on social media ASAP. So let's use [Wikipedia ](https://en.wikipedia.org/wiki/Death_of_Mahsa_Amini)content for now and then we can edit all of it later!

**Comment by AnonymousMaximus2** [comment]
I suggest you to take a look at this idea link below to develop sustainable website and grow it with efficient resources and support: 

https://github.com/WeAreMahsaAmini/Core/discussions/115

**Comment by baharehoseini** [comment]
Can anybody can put the website link here and tell what we exactly need now? I can start translating contents to different languages (Start with Italian, Spanish and Danish). Also I am ok with updating the WordPress website. Let me know what can I do please.

**Comment by syavash** [comment]
Hi, Everyone! The initial version of the website will go up by the end of today (EST).

Then we will split the different parts and get help from everyone. If you want to check the status. Please join our Discord channel: https://discord.gg/fEr6A6bK

**Reply by shahabhesaraki-dev** [reply]
Hi Syavash. 
I am a react and node developer. Please let me know if there is anything I can do.

**Comment by amirnazari96** [comment]
I've suggested having a timeline page of the Islamic Republic to show their crimes from the beginning.
https://github.com/WeAreMahsaAmini/Core/discussions/190

**Reply by amirnazari96** [reply]
btw, I'm a full-stack developer that have experience on Gatsby, React and NodeJS

**Comment by hanabarazandeh** [comment]
Hi, I am a UX/UI designer; I work with Figma and can do rapid prototyping. I have read some good suggestions about the content in this discussion. Once we agree on the content, I can help by making mockups on Figma.

**Comment by Alisideas** [comment]
I'm ready to create and design a WordPress website for wearemahsaamini.com

**Comment by ptkx132** [comment]
I can help with database and in machine learning
